compiler/src/utils/files.re

```diff
@@ -1,49 +1,88 @@
+// This file is converting strings into Fp.t and then back into Strings
+// TODO: We should consider switching to type safe Fp.t where ever filepaths are used
```

Comment:
@phated Do you mind opening an issue for this?

Reply:
Will do! Opening a few this morning